src/index.ts

```diff
@@ -1,37 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as Axe from 'axe-core';
-import { CheckMessageTransformer } from './check-message-transformer';
-import { CustomRuleConfigurations } from './custom-rule-configurations';
-import { DocumentUtils } from './document-utils';
-import { HelpUrlGetter } from './help-url-getter';
-import { MessageDecorator } from './message-decorator';
+import { ConverterOptions } from './converter-options';
 import { ResultDecorator } from './result-decorator';
 import { SarifConverter } from './sarif-converter';
 import { SarifLog } from './sarif/sarifLog';
 import { rulesWCAGConfiguration } from './wcag-mappings';
 
-export function axeToSarif(axeResults: Axe.AxeResults): SarifLog {
-    const messageDecorator = new MessageDecorator(
-        CustomRuleConfigurations,
-        new CheckMessageTransformer(),
-    );
+export { ConverterOptions } from './converter-options';
 
-    const helpUrlGetter = new HelpUrlGetter(CustomRuleConfigurations);
-    const resultDecorator = new ResultDecorator(
-        new DocumentUtils(),
-        messageDecorator,
-        ruleId => helpUrlGetter.getHelpUrl(ruleId),
-    );
+export function axeToSarif(
+    axeResults: Axe.AxeResults,
+    options?: ConverterOptions,
+): SarifLog {
+    options = options ? options : {};
 
-    resultDecorator.setWCAGConfiguration(rulesWCAGConfiguration);
+    const resultDecorator = new ResultDecorator(rulesWCAGConfiguration);
 
     const sarifConverter = new SarifConverter();
 
     // AxeResults -> ScannerResults
```

Comment:
should the comment be updated? Change `SannerResults` to `ConverterResults`? or maybe just remove the comment?

Reply:
The comment is technically still accurate; I changed the name of the ScannerOptions type, not the ScannerResults type.